test/test_playbooks/search_facts.yml

```diff
@@ -7,23 +7,11 @@
       file: server_vars.yml
   - include: tasks/search_facts.yml
     vars:
-      resource: Setting
+      resource: settings
       search: name = http_proxy
       return_length: 1
   - include: tasks/search_facts.yml
     vars:
-      resource: Subnet
+      resource: subnets
       return_length: 0
-  - name: Run with invalid resource
```

Comment:
this is currently not supported with `apypie` and will need some love there.

Reply:
OK for me.

Reply:
For the sake of clarity, how will users know what resources are valid? I assume they won't today and we can address that once we have made the required changes in apypie?

Reply:
Correct. My plan is to report back the available resources in the error message from `apypie` when a non-available one is called, and then we can restore the code we had.